README.md

```diff
@@ -5,86 +5,45 @@
 [![GoDoc](https://godoc.org/github.com/pingcap/parser?status.svg)](https://godoc.org/github.com/pingcap/parser)
 [![codecov](https://codecov.io/gh/pingcap/parser/branch/master/graph/badge.svg)](https://codecov.io/gh/pingcap/parser)
 
-## Users
-
-These projects use this parser. Please feel free to extend this list if you
-found you are one of the users but not listed here:
-
-- [pingcap/tidb](https://github.com/pingcap/tidb)
-- [XiaoMi/soar](https://github.com/XiaoMi/soar)
-- [XiaoMi/Gaea](https://github.com/XiaoMi/Gaea)
-- [sql-machine-learning/sqlflow](https://github.com/sql-machine-learning/sqlflow)
-
-## How to use it
-
-```go
-import (
-	"fmt"
-	"github.com/pingcap/parser"
-	_ "github.com/pingcap/tidb/types/parser_driver"
-)
-
-// This example show how to parse a text sql into ast.
-func example() {
-
-	// 0. make sure import parser_driver implemented by TiDB(user also can implement own driver by self).
-	// and add `import _ "github.com/pingcap/tidb/types/parser_driver"` in the head of file.
-
-	// 1. Create a parser. The parser is NOT goroutine safe and should
-	// not be shared among multiple goroutines. However, parser is also
-	// heavy, so each goroutine should reuse its own local instance if
-	// possible.
-	p := parser.New()
+The goal of this project is to build a Golang parser that is fully compatible with MySQL syntax, easy to extend, and high performance. Currently, features supported by parser are as follows:
 
-	// 2. Parse a text SQL into AST([]ast.StmtNode).
-	stmtNodes, _, err := p.Parse("select * from tbl where id = 1", "", "")
+- Highly compatible with MySQL: it supports almost all features of MySQL. For the complete details, see [parser.y](https://github.com/pingcap/parser/blob/master/parser.y) and [hintparser.y](https://github.com/pingcap/parser/blob/master/hintparser.y).
+- Extensible: adding a new syntax requires only a few lines of Yacc and Golang code changes. As an example, see [PR-680](https://github.com/pingcap/parser/pull/680/files).
+- Good performance: the parser is generated by goyacc in a bottom-up approach. It is efficient to build an AST tree with a state machine.
 
-	// 3. Use AST to do cool things.
-	fmt.Println(stmtNodes[0], err)
-}
-```
+## Documentation
 
-See [https://godoc.org/github.com/pingcap/parser](https://godoc.org/github.com/pingcap/parser)
+Please read the [quickstart](https://github.com/pingcap/parser/blob/master/docs/quickstart.md).
 
-## How to update parser for TiDB
+## Future
```

Comment:
maybe `Feature`? BTW, I think we should introduce this section before `Documentation`

Reply:
This section really menas "Future" as in "things to do in the future".

Reply:
OK 😂